### ISSUE TITLE
Default Vimeo embed code/regex is out of date

### ISSUE THREAD

**Issue by sambuev** [vichan-devel/vichan#260]
Hello, can you please update the embed code from vimeo? 
If you can, can you please take the code from this site: 

http://embedresponsively.com/

I don't know how to do it, if you can update the core of vichan it's going to help many users!
Thank you.

**Comment by goosehub** [comment]
Don't know if it helps, but I wrote a vimeo embed regex as part of my url into embed library.

https://github.com/goosehub/embedica/

```
function embedica_convert_vimeo(input) {
  var pattern = /(?:http?s?:\/\/)?(?:www\.)?(?:vimeo\.com)\/?(\S+)/g;
  if (pattern.test(input)) {
    var replacement = '<span class="embedica_vimeo embedica_element"><iframe src="//player.vimeo.com/video/$1" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe></span>';
    var input = input.replace(pattern, replacement);
  }
  return input;
}
```

**Comment by ctrlcctrlv** [comment]
Fixed in #262
